decidim-core/lib/decidim/core/test/factories.rb

```diff
@@ -51,83 +51,6 @@
     official_url { Faker::Internet.url }
   end
 
-  factory :participatory_process, class: Decidim::ParticipatoryProcess do
```

Comment:
What about factories for categories and moderations? You are moving the models in the previous commit.